background.ts

```diff
@@ -542,15 +542,19 @@ ipcMainProxy.on('k8s-progress', () => {
 });
 
 ipcMainProxy.handle('service-fetch', (_, namespace) => {
+  console.debug(`service-fetch: ${ namespace }`);
```

Comment:
Same comment as above - this is unnecessary right?

Reply:
removed